reframe/core/pipeline.py

```diff
@@ -777,6 +785,9 @@ def _rfm_init(self, name=None, prefix=None):
         # True only if check is to be run locally
         self.local = False
 
+        # True only if check is to be built locally
```

Comment:
I don't think that this comment is needed. For `local` it has a bit more meaning, but most probably it's there before even documenting the fields.